lib/vicadmin/validate.go

```diff
@@ -55,6 +54,9 @@ type Validator struct {
 	DockerPort       string
 	VCHStatus        template.HTML
 	VCHIssues        template.HTML
+	VCHReachable     bool
+	VCHUnreachable   bool
```

Comment:
`VCHUnreachable` is redundant. 

I tried using an `else` block [here](https://github.com/vmware/vic/pull/3106/files#diff-7372f4b2bbd435e1a9f6b78eeb5f04f0R105) to show the error string under the log bundle link, but the template couldn't be parsed. I'll modify the template logic to get rid of `VCHUnreachable` if we can.